### ISSUE TITLE
Pylance is not working on vscode.dev/github.dev 

### ISSUE THREAD

**Issue by luabud** [microsoft/pylance-release#2235]
## Environment data

-   Language Server version: v2022.1.0
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): N/A

## Expected behaviour

When I open a Python file on the browser, I get semantic highlight and simply auto completions for Python files 

## Actual behaviour

Pylance seems to not be working at all when in vscode.dev/github.dev

## Logs

```
[Info  - 9:25:59 AM] Pylance language server 2022.1.0 (pyright 5829dcbb) starting
[Info  - 9:25:59 AM] Server root directory: /
[Info  - 9:25:59 AM] No configuration file found.
[Info  - 9:25:59 AM] No pyproject.toml file found.
[Warn  - 9:25:59 AM] stubPath /pvscc-logo/typings is not a valid directory.
Search paths for /pvscc-logo
  /typeshed-fallback/stdlib
  /pvscc-logo
  /pvscc-logo/typings
  /typeshed-fallback/stubs/...
  /bundled/stubs
[Info  - 9:25:59 AM] Searching for source files
[Error - 9:25:59 AM] File or directory "/pvscc-logo" does not exist.
[Info  - 9:25:59 AM] No source files found.

```

## Code Snippet / Additional information

```python
XXX
```

**Comment by heejaechang** [comment]
Thank you for the report.

it looks like it got broke.

![image](https://user-images.githubusercontent.com/1333179/148814502-706cba6d-7f4c-4095-a1fb-ba9f0eb307b3.png)

taking a look.

**Comment by luabud** [comment]
thanks for looking into it!

**Comment by bschnurr** [comment]
This issue has been fixed in version 2022.1.1, which we've just released. You can find the changelog here:  [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202211-12-january-2022)
